views/Navbar.tsx

```diff
@@ -4,24 +4,54 @@ import { AiOutlineMenu, AiOutlineClose } from 'react-icons/ai';
 import LoginLogout from "../components/LoginLogout";
 import chromeLogo from '../public/chrome-logo.png'
 import Image from 'next/image';
+import { rudderEventMethods } from '../utils/rudderstack_initialize';
+import { v4 as uuidv4 } from 'uuid';
 
-const Navbar = (props: { ctaLink: string, transparent: boolean }) => {
-  const [showNavbar, setShowNavbar] = useState(false);
-  const [scrollDown, setScrollDown] = useState(props.transparent);
-  const changeNavbar = () => {
-    setShowNavbar(!showNavbar);
-  };
-  useEffect(() => {
-    const changeColor = () => {
-      if (window.scrollY >= 90) {
-        setScrollDown(true);
-      } else {
-        setScrollDown(false);
-      }
-    };
-    window.addEventListener('scroll', changeColor);
-  }, []);
 
+	const Navbar = (props: { ctaLink: string, transparent: boolean }) => {
+	const [showNavbar, setShowNavbar] = useState(false);
+	const [scrollDown, setScrollDown] = useState(props.transparent);
+	const changeNavbar = () => {
+		setShowNavbar(!showNavbar);
+	};
```

Comment:
Do we really need all this?

Reply:
I don't know. Just fixed the indentation